pkgs/applications/networking/instant-messengers/jami/daemon.nix

```diff
@@ -67,7 +67,11 @@ let
 
       src = fetchFromGitHub src-args;
 
-      patches = old.patches ++ (map (x: patch-src + x) (readLinesToList ./config/pjsip_patches));
+      patches = map (x: patch-src + x) (readLinesToList ./config/pjsip_patches);
+
+      meta = {
+        knownVulnerabilities = [ "CVE-2022-39269" "CVE-2022-39244" ];
+      } // old.meta;
```

Comment:
According to upstream, jami is not affected by this CVE. Not sure if we can bump the version to `513a3f14c44b2c2652f9219ec20dea64b236b713` where the patch is applied.

Reply:
I think we need to bump jami to a version after https://git.jami.net/savoirfairelinux/jami-daemon/-/commit/120eb26bcce66dd5edf3c517693f79337b11b53c, not for CVE-2022-39269 & CVE-2022-39244, but for the issues addressed in the pjsip 2.12.1 release (https://github.com/pjsip/pjproject/releases/tag/2.12.1) which go well beyond the pjmedia component.

But we should do that in unstable first and port it back to 22.05 in a separate PR. For now I'll remove the `jami` section from this PR.

Reply:
Ok, I did that in https://github.com/NixOS/nixpkgs/pull/198068.